runtime/account_keys_test.go

```diff
@@ -193,43 +193,6 @@ func TestRuntimeAccountKeyConstructor(t *testing.T) {
 	assert.Contains(t, err.Error(), "cannot find variable in this scope: `AccountKey`")
 }
 
-func TestRuntimeImportAccountAPITypes(t *testing.T) {
```

Comment:
Same scenario is already covered in `TestScriptParameterTypeValidation`. So this is a redundant test case.